### PR TITLE
Sync with latest develop version for a new release

### PR DESCRIPTION
**Pull request by jjos2372**

### CODE REVIEW

File: openapi/signum-api.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "openapi": "3.0.3",
   "info": {
     "title": "Signum Node API",
-    "version": "3.4.4",
+    "version": "3.5.0",
```
Comment (frankTheTank72) [inline openapi/signum-api.json:5]: Not 3.5.1 ?

@ohager

**Review by frankTheTank72 — COMMENTED** [review]
Found two small issues - see questions

File: openapi/signum-api.json (hunk 1 above)
Comment (ohager) [inline openapi/signum-api.json:5]:
```suggestion
    "version": "3.5.1",
```

**Review by ohager — CHANGES_REQUESTED** [review]
update version

**Review by ohager — APPROVED** [review]

**Review by frankTheTank72 — APPROVED** [review]

**Pull request merged**